etc/adminhtml/system.xml

```diff
@@ -95,6 +95,11 @@
                         <config_path>payment/boltpay/order_management</config_path>
                         <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                     </field>
+                    <field id="always_present_checkout" translate="label" type="select" sortOrder="83" showInDefault="0" showInWebsite="0" showInStore="0">
+                        <label>(BETA) Enable Always Present Checkout Button</label>
```

Comment:
@njfaries An idle thought here - maybe we should consider having a section for developer options/beta features, similar to what we have in the admin dash. Then our beta features aren't in the middle of the regular config settings.

Out of scope for this PR but it had me thinking.